packages/compass-collection/src/components/mock-data-generator-modal/script-screen.tsx

```diff
@@ -100,9 +131,22 @@ const ScriptScreen = () => {
           In the directory that you created, create a file named
           mockdatascript.js (or any name you&apos;d like).
         </Body>
-        {/* TODO: CLOUDP-333860: Hook up to the code generated as part script generation */}
-        <Code copyable language={Language.JavaScript}>
-          TK
+        {!scriptResult.success && (
+          <Banner variant="danger">
+            <strong>Script Generation Failed:</strong> {scriptResult.error}
+            <br />
+            Please go back to the start screen to re-submit the collection
+            schema.
+          </Banner>
+        )}
+        <Code
+          copyable={scriptResult.success}
+          language={Language.JavaScript}
+          style={{ maxHeight: '230px', overflowY: 'auto' }}
```

Comment:
That's an unexpected usage of `style` property, this is static, so can be defined as a className outside of render with `css({})`